cfme/tests/automate/custom_button/test_service_objects.py

```diff
@@ -713,3 +716,67 @@ def test_custom_button_open_url_service_obj(objects, button_group):
         1550002
     """
     pass
+
+
+@pytest.fixture(params=["Service", "Other"], scope="module")
+def unassigned_btn_setup(request, appliance, provider, service):
+    if request.param == "Service":
+        btn_type = request.param
+        obj = MyService(appliance, name=service.name)
+        destinations = [ViaUI, ViaSSUI]
+    else:
+        btn_type = "Provider"
+        obj = provider
+        destinations = [ViaUI]
+
+    gp = appliance.collections.button_groups.instantiate(
+        text="[Unassigned Buttons]", hover="Unassigned buttons", type=btn_type
+    )
+    yield obj, gp, destinations
+
+
+@pytest.mark.provider([VMwareProvider], override=True, scope="module", selector=ONE)
+def test_custom_button_unassigned_behavior_objs(
+    appliance, setup_provider, unassigned_btn_setup, request
+):
+    """ Test unassigned custom button behavior
+
+    Note: Service unassigned custom button should display on SSUI but not OPS UI.
+    For other than service objects also fallows same behaviour l.e. not display on OPS UI.
+
+    Polarion:
+        assignee: ndhandre
+        initialEstimate: 1/6h
+        caseimportance: medium
+        caseposneg: positive
+        testtype: functional
+        startsin: 5.8
+        casecomponent: CustomButton
+        testSteps:
+            1. Create unassigned custom button on service and one other custom button object.
```

Comment:
I created one button under "Unassigned" for Service and it shows up in OSP UI service .
SO the behaviour is same for OSP and SUI right? 
Let me know if I am checking at the wrong place.
Appliance : https://10.8.196.131 , check button on service "name"

Reply:
https://bugzilla.redhat.com/show_bug.cgi?id=1709870

For more info
https://bugzilla.redhat.com/show_bug.cgi?id=1653195

Reply:
I'm waiting for the final call from PM or some Developer on BZ.
Just need to change one line in assert and test will be ready to merge.

Reply:
@sshveta as per talk with Harpreet it should not but still no PM ack on behavior. I  would like to go with this for now. As changes appear I will take care of those.